payments/tests/test_payment_api.py

```diff
@@ -0,0 +1,81 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from books.models import Book
+from borrowings.models import Borrowing
+from payments.models import Payment
+
+PAYMENT_URL = reverse("payments:payment-list")
+
+User = get_user_model()
+
+
+def get_detail(payment_id: int) -> str:
+    return reverse("payments:payment-detail", args=[payment_id])
+
+
+class PaymentAPITests(TestCase):
```

Comment:
You can also add tests for checking that `PaymentRetrieveSerializer `returns in get detail